full_assets/ElephantWithMonochord/Readme.md

```diff
@@ -0,0 +1,46 @@
+# Songs of Cultures – Vietnamese Elephant with Monochord
+
+## Video
+
+screenshots/20220604-video.mp4  
+_Video with Audio_
+
+## Screenshot
+
+![omniverse](screenshots/20220604-omniverse.png)  
+_Oniverse CREATE with skeleton hierarchies and UsdAudio_
+
+## Augmented Reality
+
+Open this page on iOS and click on the image:  
```

Comment:
I downloaded the asset, and hit spacebar on a mac ~ it played properly with sound and without transform or rendering errors. Do we want to say anything about mac quicklook?

Reply:
Added a note that this should also work on MacOS QuickLook
(side note: I don't think the Mac and iOS versions are identical)